csrc/scheduler/reduction_utils.cpp

```diff
@@ -740,9 +733,9 @@ TensorView* sortAndRFactor(TensorView* reference_tv) {
     // unswitch dim.
     if (!(id->getParallelType() == ParallelType::Unswitch &&
           id->extent()->isOneInt())) {
-      rfactor_axes_no_unswitch.emplace_back(axis_i);
+      rfactor_axes_no_unswitch.push_back(axis_i);
```

Comment:
I tend to use push_back for trivial types. emplace_back has a richer semantic, e.g., creating an object from the arg, which can be confusing. 

https://abseil.io/tips/112